terraform/00_from_local/provider.tf

```diff
@@ -9,9 +9,10 @@ terraform {
 }
 
 provider "openstack" {
-  user_name = "plgppycia"
+  user_name = var.openstack_username
+  password = var.openstack_password
```

Comment:
Aren't evn variables better for our case? Was there an issue with them?

Reply:
They should be better, didn't look into the topic deeper at the moment of writing it. Just wanted to avoid hardcoding sensitive data. Will use env vars.

Reply:
https://github.com/k8loud/kubernetes-lab-setup/issues/18